### DISCUSSION TITLE
What is the Bookmarks (edited)

### DISCUSSION THREAD

**Discussion by Monirzadeh** [koreader/koreader#11650] [Q&A] [answered]
Hi
in koreader if you go to top menu > bookmarks > and tap on hamburger menu in top corner you see an option `filter by edited highlighted text` and you see a menu name `Bookmarks (edited)`
what is it exactly? it is empty for me and i don't understand the functionality of that.

**Comment by Frenzie** [comment, marked as answer]
It's for if you long-press a highlight and use "edit text" to add some extra notes or some such.

**Reply by Monirzadeh** [reply]
so it is some kind of note but not define it as note right? it is a little weird to me. do you know why someone need this? what is the advantage of this over note?

**Reply by poire-z** [reply]
It's been requested with insistance :) See #9900.
Possibly more needed for people reading PDF where OCR extracting text may have errors, so users can correct/rewrite the extracted text so it looks nice in that menu and exported highlights.

**Reply by Monirzadeh** [reply]
Oh :)
Thanks it make sense now.
